package.json

```diff
@@ -6,14 +6,14 @@
   "scripts": {
     "compile": "tsc",
     "copy-graphql": "mkdir ./dist/schemas && cp ./src/schemas/*.graphql ./dist/schemas/",
-    "build": "rm -rf ./dist && npm run compile",
     "generate": "graphql-codegen",
-    "dev": "ts-node-dev --respawn ./src/index.ts",
+    "lint": "npx eslint . --ignore-pattern dist/ --ignore-pattern src/types.ts",
     "test": "jest --coverage",
-    "test-tsc": "jest -c ./jest.tsc.config.js",
-    "lint": "npx eslint .",
+    "build": "rm -rf ./dist && npm run lint && npm run generate && npm run test && npm run compile",
```

Comment:
@briri - I'm wondering if we should exclude "`npm run lint`" and "`npm run test`" from "`build`". In the code pipeline, we will probably only want to run "build" after the linting and tests pass.

Reply:
Yes. You are right. I will remove